File: frontend/src/models/EngineerUserAbility.ts
export default interface EngineerUserAbility {
  id: number;
  engineerUserId: number;
  projectScore: number;
  repositoryScore: number;
  commitScore: number;
  pullreqScore: number;
  issueScore: number;
  speedScore: number;
  createdAt: Date;
}

File: backend/model/engineer_user_ability.go
package model

import "time"

type EngineerUserAbility struct {
	ID              uint64
	EngineerUserID  uint64
	ProjectScore    uint64
	RepositoryScore uint64
	CommitScore     uint64
	PullreqScore    uint64
	IssueScore      uint64
	SpeedScore      uint64
	CreatedAt       time.Time
}
